src/NHSD.BuyingCatalogue.Ordering.Api/Controllers/ServiceRecipientsSectionController.cs

```diff
@@ -49,11 +49,13 @@ public async Task<ActionResult<ServiceRecipientsModel>> GetAllAsync(string order
             var serviceRecipients =
                 (await _serviceRecipientRepository.ListServiceRecipientsByOrderIdAsync(orderId)).ToList();
 
-            var recipientModelList = serviceRecipients.Select(recipient => new ServiceRecipientModel
-            {
-                OdsCode = recipient.OdsCode,
-                Name = recipient.Name
-            }).ToList();
+            var recipientModelList = serviceRecipients
+                .Select(recipient => new ServiceRecipientModel
+                {
+                    OdsCode = recipient.OdsCode,
+                    Name = recipient.Name
+                }).OrderBy(serviceRecipientModel => serviceRecipientModel.Name)
```

Comment:
Can you please put this on a new line at the same level as the .Select and .ToList() please?

Reply:
👍